src/pages/asset.tsx

```diff
@@ -0,0 +1,133 @@
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import { Box, IconButton, Typography, useTheme } from '@mui/material';
+import type { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { AssetGraphBoxBorrow } from '../components/asset/AssetGraphBoxBorrow';
+import { AssetGraphBoxSupply } from '../components/asset/AssetGraphBoxSupply';
+import { Divider } from '../components/common/Divider';
+import { ReserveDropdown } from '../components/common/ReserveDropdown';
+import { Row } from '../components/common/Row';
+import { Section, SectionSize } from '../components/common/Section';
+import { Skeleton } from '../components/common/Skeleton';
+import { PoolHeader } from '../components/pool/PoolHeader';
+import { useSettings, ViewType } from '../contexts';
+import { usePool, usePoolOracle } from '../hooks/api';
+import { toPercentage } from '../utils/formatter';
+import { getTokenLinkFromReserve } from '../utils/token';
+const Asset: NextPage = () => {
+  const router = useRouter();
+  const theme = useTheme();
+  const { viewType } = useSettings();
+
+  const { poolId, assetId } = router.query;
+  const safePoolId = typeof poolId == 'string' && /^[0-9A-Z]{56}$/.test(poolId) ? poolId : '';
+  const safeAssetId = typeof assetId == 'string' && /^[0-9A-Z]{56}$/.test(assetId) ? assetId : '';
+  const { data: pool } = usePool(safePoolId);
+  const { data: poolOracle, isError: isOracleError } = usePoolOracle(pool);
+  const reserve = pool?.reserves.get(safeAssetId);
+  const hasData = pool && poolOracle && reserve;
+  const link = getTokenLinkFromReserve(reserve);
+
+  return (
+    <>
+      <Row>
+        <Section
+          width={SectionSize.FULL}
+          sx={{ marginTop: '12px', backgroundColor: theme.palette.menu.opaque }}
+        >
+          <PoolHeader name={pool?.config.name ?? 'Unknown'} sx={{ paddingLeft: '12px' }} />
+        </Section>
+      </Row>
+      <Row sx={{ justifyContent: 'flex-start', alignItems: 'center' }}>
+        <Section width={SectionSize.FULL} sx={{ marginBottom: '12px' }}>
+          <ReserveDropdown action="asset" poolId={safePoolId} activeReserveId={safeAssetId} />
+        </Section>
+        <IconButton
+          onClick={() => window.open(link, '_blank')}
+          size="small"
+          sx={{
+            color: theme.palette.text.secondary,
+          }}
+        >
+          <OpenInNewIcon fontSize="inherit" />
+        </IconButton>
```

Comment:
section full is intended to take up the whole row, not have something rendered next to it.

This being said, I think we need to update this to be more like the pool dropdown on the markets page.

That is:
```
[ Reserver Dropdown ]  [Explorer ]
```

The current link looks very out of place. For mobile, we could replace the `Explorer` button with a smaller button that just contains the external link icon, and retains the background / box shadow

Reply:
I still don't know if I like the look of this tbh... I think just having the `asset header` with a `go back button` looks a lot cleaner